dkms.in

```diff
@@ -287,6 +287,9 @@ setup_kernels_arches()
                 fi
             fi
         fi
+        if [[ ! $arch ]]; then
+            die 12 $"Could not determine architecture."
+        fi
     fi
```

Comment:
Is there a real world use-case when this can happen or is this a case of defensive programming? In either case, please add some details in the commit message.

Reply:
it happened to one user: https://bugs.debian.org/1036033 but is very unlikely to happen more frequently, therefore I'm not trying to add a testcase for it (seems non-trivial)